libc/src/search/hcreate.cpp

```diff
@@ -14,6 +14,12 @@
 
 namespace LIBC_NAMESPACE {
 LLVM_LIBC_FUNCTION(int, hcreate, (size_t capacity)) {
+  // We follow FreeBSD's implementation here. If the global_hash_table is
```

Comment:
Since this case is not specified in the standards, can you document this behavior, maybe in: https://github.com/llvm/llvm-project/blob/main/libc/docs/dev/undefined_behavior.rst ?

Reply:
Added